assets/styles/community.scss

```diff
@@ -0,0 +1,79 @@
+.post-wall-container {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+
+    .post-wall-head {
+        background-color: $primary-color;
+        border-bottom-left-radius: 50% 20%;
+        border-bottom-right-radius: 50% 20%;
+
+        h1 {
+            font-size: 2.5rem; 
+            color: white;
+            margin-top: 1rem;
+        }
+
+        li {
+            color: white;
+            margin-top: 1.5rem;
+        }
+
+        .post-create {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            margin-top: 1.5rem;
+
+            h2 {
+                align-self:flex-start;
+                margin-left: 12vw;
```

Comment:
looks like you're searching for align-self: center, aren't you?

Reply:
no :)